build_targets

```diff
@@ -19,3 +19,4 @@ sweet user yes yes
 vayu user yes yes
 venus userdebug yes yes
 zippo user yes yes
+lunaa userdebug yes yes
```

Comment:
move to proper place (alphabetically) 

invalid branch
https://github.com/crdroidandroid/android_device_oneplus_sm8350-common/blob/14.0-lunaa/crdroid.dependencies#L18#L22 

invalid repo confguration
https://github.com/crdroidandroid/android_device_oneplus_sm8350-common/blob/14.0-lunaa/crdroid.dependencies#L24